pkg/utils/k8s/k8sutils.go

```diff
@@ -90,6 +91,7 @@ func TerminatingPods(pods []corev1.Pod) []corev1.Pod {
 	return terminating
 }
 
+// RunningPods filters pods for Pods are in running (and not terminating).
```

Comment:
nit: feels like something is missing
```suggestion
// RunningPods filters pods for Pods that are in the running phase (and not terminating).
```